cypress/e2e/pages/batches.pages.js

```diff
@@ -62,7 +62,8 @@ function executeTransaction() {
 }
 
 function addToBatchButton() {
-  cy.contains(addToBatchBtn).should('be.visible').and('not.be.disabled').click()
+  cy.wait(1000)
```

Comment:
I don't think this timeout is necessary. The button is rendered immediately, it's just not in the viewport.